oak_runtime/src/node/http/server.rs

```diff
@@ -358,6 +362,21 @@ impl HttpRequestHandler {
     }
 }
 
+/// Check if the request contains a `TRANSFER_ENCODING` header, and reject the request in that case
+/// by returning an error.
```

Comment:
This can be removed once https://rustsec.org/advisories/RUSTSEC-2021-0020.html does not apply any more, right? which seems to be fixed in some newer versions already. Could you add a TODO and create an issue to remove the workaround?

Reply:
Yes, it is already fixed in `hyper:0.14.4`, but `tonic` is not updated. Do we have an issue for tracking tonic's version? Because we are already behind.

Reply:
I don't think we do, we just update manually or when dependabot decides to :)

Reply:
Created an issue for it to be able to add a TODO comment.